src/app/redux/MarketReducer.js

```diff
@@ -1,40 +1,29 @@
 import {Map} from 'immutable';
-import createModule from 'redux-modules';
-
-
-export default createModule({
-    name: 'market',
-    initialState: Map({status: {}}),
-    transformations: [
-        {
-            action: 'RECEIVE_ORDERBOOK',
-            reducer: (state, action) => {
-                return state.set('orderbook', action.payload);
-            }
-        },
-        {
-            action: 'RECEIVE_TICKER',
-            reducer: (state, action) => {
-                return state.set('ticker', action.payload);
-            }
-        },
-        {
-            action: 'RECEIVE_OPEN_ORDERS',
-            reducer: (state, action) => {
-                return state.set('open_orders', action.payload);
-            }
-        },
-        {
-            action: 'RECEIVE_TRADE_HISTORY',
-            reducer: (state, action) => {
-                return state.set('history', action.payload);
-            }
-        },
-        {
-            action: 'APPEND_TRADE_HISTORY',
-            reducer: (state, action) => {
-                return state.set('history', [...action.payload, ...state.get('history')]);
-            }
-        }
-    ]
-});
+
+const defaultState = Map({status: {}});
+
+export default function reducer(state = defaultState, action) {
+    const payload = action.payload;
+
+    if (action.type === 'market/RECEIVE_ORDERBOOK') {
+        return state.set('orderbook', payload);
+    }
+
+    if (action.type === 'market/RECEIVE_TICKER') {
+        return state.set('ticker', payload);
+    }
+
+    if (action.type === 'market/RECEIVE_OPEN_ORDERS') {
```

Comment:
I recommend creating a few `constants` files that contains these strings. Without action creators or reducer tests these are _very_ susceptible to typos.